packages/dds/tree/src/test/feature-libraries/sequence-field/testEdits.ts

```diff
@@ -57,12 +58,19 @@ function createReviveChangeset(
 	startIndex: number,
 	count: number,
 	detachedBy: RevisionTag,
+	reviver: NodeReviver,
```

Comment:
Since most tests will want to use `fakeRepair`, I think it would be best to just default to that and move this parameter to be in last position. Same for `createIntentionalReviveChangeset`.